src/app/(guest)/profile/[token]/page.tsx

```diff
@@ -57,7 +57,7 @@ const profile: React.FC<ProfileProps> = async ({ params }) => {
         Os teus <span className="text-primary">exames</span>
       </p>
 
-      <PreviousExamsTable user={user} token={params.token} />
+      <PreviousExamsTable nTotalAnswers={user.answers.length} token={params.token} />
```

Comment:
As answers are a separate route now this prop doesn't make sense. The fetch occurs inside the component